### PR TITLE
feat(examples): Add Scaleway Golang example

### PR DESCRIPTION
**Pull request by abarbare**
This PR adds a new Golang example to use the Scaleway provider.

### CODE REVIEW

File: examples/go/scaleway/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Scaleway example
+
+- Reserve an IP address
+- Create a Compute Instance with the reserved IP address
+
+Don't forget to export your credentials as environement variables
```
Comment (ansgarm) [inline examples/go/scaleway/README.md:6]:
```suggestion
Don't forget to export your credentials as environment variables
```

File: examples/go/scaleway/cdktf.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "language": "go",
+    "app": "go run main.go",
+    "codeMakerOutput": "generated",
+    "projectId": "8acbba0c-66a6-471b-bd44-9641bbc0f3a3",
+    "terraformProviders": [
+        "scaleway/scaleway@>= 0.13"
```
Comment (ansgarm) [inline examples/go/scaleway/cdktf.json:7]: Is there a reason you did not depend on the latest one? I couldn't even find a 0.13 version and it seems that even 1.x is already two years old 🙂

**Review by ansgarm — COMMENTED** [review]
Nice work @abarbare!

File: examples/go/scaleway/README.md (hunk 1 above)
Comment (abarbare) [reply examples/go/scaleway/README.md]: Thanks, updated 🙂

File: examples/go/scaleway/cdktf.json (hunk 2 above)
Comment (abarbare) [reply examples/go/scaleway/cdktf.json]: Indeed, I copy/pasted the version I found on terraform documentation but your right, 2.2.0 version is out now :)

**Review by ansgarm — APPROVED** [review]

**Pull request merged**